service/awsconfig/v7/drainer_resource_set.go

```diff
@@ -40,21 +41,21 @@ func NewDrainerResourceSet(config DrainerResourceSetConfig) (*framework.Resource
 
 	var err error
 
-	//var lifecycleResource framework.Resource
-	//{
-	//	c := lifecycle.ResourceConfig{
-	//		Clients:   config.GuestAWSClients,
-	//		Logger:    config.Logger,
-	//	}
-	//
-	//	lifecycleResource, err = lifecycle.NewResource(c)
-	//	if err != nil {
-	//		return nil, microerror.Mask(err)
-	//	}
-	//}
+	var lifecycleResource framework.Resource
+	{
+		c := lifecycle.ResourceConfig{
+			Clients: config.GuestAWSClients,
+			Logger:  config.Logger,
+		}
+
+		lifecycleResource, err = lifecycle.NewResource(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```

Comment:
The lifecycle resource got activated now.

Reply:
It is noop now, isn't it?

Reply:
It logs a message when it is executed. Or what do you mean?

Reply:
I wanted to make sure lifecycle resource doesn't do anything